### ISSUE TITLE
School/Work Information

### ISSUE THREAD

**Issue by pqhf5kd** [monicahq/monica#2082]
Currently we have Work Infomation on the left and New Job, New School & Study Aboard under Life Events.

Could the information on the left be automatically updated when a new Life Event is added?

**Comment by djaiss** [comment]
Yep. I have done the Life Events in such a way that this will be possible in the future.
